packages/design-system/src/components/Chip.tsx

```diff
@@ -80,7 +58,16 @@ export const Chip = ({
         },
         className,
       )}
-      icon={icon ? <Icon className="!m-0" name={icon} /> : undefined}
+      icon={
+        icon ? (
+          <Icon
+            className="!m-0"
+            name={icon}
+            size={iconSize ?? 'small'}
```

Comment:
`<Icon />` component's size default value is `medium`.
This change makes all Chip now have a small `Icon` by default.
Is that expected? Or maybe the `Chip` was always provided an `Icon` tsx element on invocation?